test/test_package.py

```diff
@@ -118,6 +118,16 @@ def test_init_dependency(self):
         dep = Dependency('foo', condition='foo <= bar or bar >= baz')
         self.assertFalse(dep.evaluate_condition({}))
 
+    def test_dependency_repr(self):
+        dep = Dependency('foo', condition='$foo == 2')
+        assert repr(dep) == "Dependency(name='foo', condition='$foo == 2')"
+        self.assertTrue(dep.evaluate_condition({'foo': 2}))
+        assert repr(dep) == "Dependency(name='foo', condition='$foo == 2', evaluated_condition=True)"
+
+        dep = Dependency('foo', condition='$foo == 2')
```

Comment:
Nitpick: is this line really necessary? Can't it instead use the already existing `dep`?

Reply:
You are right, you can re-evaluate the dependency condition. I've updated the PR